nodejs/image.bzl

```diff
@@ -99,8 +99,27 @@ def _npm_deps_runfiles(dep):
             depsets.append(pkg[NpmPackageInfo].sources)
     return depset(transitive = depsets)
 
+def _npm_deps_symlinks(reference_dir, binary):
```

Comment:
I'm open to feedback of the implementation of this. I'm not sure if this is the best place for it, and I don't like the search logic to find a "FileWrite" command that writes a ".module_mappings.json" file. But I'm new to Starlark, and this works, so very open to feedback on the best way to write it and how to add tests :)